Source/WebCore/Modules/cookie-store/CookieStore.idl

```diff
@@ -29,8 +29,8 @@ typedef sequence<CookieListItem> CookieList;
 
 [
     ActiveDOMObject,
-    EnabledBySetting=CookieStoreAPIEnabled,
-    Exposed=(Window),
+    EnabledBySetting=CookieStoreAPIEnabled&CookieStoreAPIServiceWorkerEnabled,
```

Comment:
We should drop the `&CookieStoreAPIServiceWorkerEnabled` here. We want to expose this to Window even if the CookieStoreAPIServiceWorkerEnabled setting is not enabled.